### ISSUE TITLE
not marking movies as watched after scrobbling

### ISSUE THREAD

**Issue by carlosmg2** [trakt/Plex-Trakt-Scrobbler#289]
have an issue with the latest versions of this plugin. scrobbling works nice. after finishing watching tv series episodes they are marked as watched on trakt. but movies don't. movies ar scrobbled nice during playback but at the end they are not marked as watched on trakt.
anyone have this issue?

**Comment by fuzeman** [comment]
Could you post the latest plugin log file after watching a movie? see https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files for the location of the log files.

Also, could you let me know which Plex client you are using? (PHT, DLNA, Plex/Web, etc..)

**Comment by carlosmg2** [comment]
sorry for the missing info. I'm using Open PHT 1.5.1 stable. PMS 0.9.15.6, On Windows 10 x64.
I attach some logs:

[Movie1] On log.2 i watched a movie from 2016-03-06 15:40  to 17:40. As the plugin didn't mark as watched on trakt web I marked manually as watched on web later.

[com.plexapp.plugins.trakttv.log.2.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/163151/com.plexapp.plugins.trakttv.log.2.txt)

[Movie2]
on log.3 I watched a movie from 2016-03-05 22:00 to 2016-03-05 23:59.
As it didn't mark as watched on trakt web I marked manually as watched on web later.

[com.plexapp.plugins.trakttv.log.3.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/163159/com.plexapp.plugins.trakttv.log.3.txt)


If this isn't useful I could activate debug level logs and send other day.

**Comment by fuzeman** [comment]
This message is likely related to the issue:
```
Missing session key, unable to fetch session details
```

I'll do some tests with OpenPHT tonight, there could be an issue retrieving watching activity for that client.

**Comment by fuzeman** [comment]
Haven't been able to find any issues with OpenPHT, could you post a log file with debug logging enabled?

Also, which metadata agent are you using for Movie libraries? the plugin requires specific support added for each metadata agent.

**Comment by carlosmg2** [comment]
The metadata agent es "The Movie Database".

And now weird things:
- after enabling debug logs i watched 2 films yesterday and they scrobbled and marked as watched nicely.
- now i have just watched a film and it didn't srobble

here are the logs. I started watching it around 15h 

[com.plexapp.plugins.trakttv.log.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/170484/com.plexapp.plugins.trakttv.log.txt)
[com.plexapp.plugins.trakttv.log.1.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/170485/com.plexapp.plugins.trakttv.log.1.txt)
[com.plexapp.plugins.trakttv.log.2.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/170486/com.plexapp.plugins.trakttv.log.2.txt)
[com.plexapp.plugins.trakttv.log.3.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/170487/com.plexapp.plugins.trakttv.log.3.txt)
[com.plexapp.plugins.trakttv.log.4.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/170488/com.plexapp.plugins.trakttv.log.4.txt)

**Comment by fuzeman** [comment]
Looks like sessions aren't being updated correctly when the media changes, this means incorrect "stop" events are sent to Trakt.tv:
```
Sending action u'scrobble/stop' (account: <Account name: [...]>, interface: u'scrobble', method: u'stop')
request: {u'movie': {u'title': u'La gran revancha', u'ids': {u'tmdb': 64807}, u'year': 2013}, u'progress': 49.53}
```

Will try have this fixed tomorrow :smile:

**Comment by fuzeman** [comment]
Should be fixed in this version of the plugin: https://github.com/fuzeman/Plex-Trakt-Scrobbler/archive/hotfix/1.0.1.2.zip

Post the latest plugin log file again if you notice this issue is still happening.

**Comment by fuzeman** [comment]
Fix released in [v1.0.1.2](https://github.com/trakt/Plex-Trakt-Scrobbler/releases/tag/v1.0.1.2)
